test/spec/GeoExt/data/store/Features.test.js

```diff
@@ -101,6 +101,30 @@ describe('GeoExt.data.store.Features', function() {
         });
     });
 
+    describe('#getByFeature', function() {
+        var coll,
+            store,
+            feature;
+        beforeEach(function() {
+            feature = new ol.Feature();
+            coll = new ol.Collection();
+            coll.push(feature);
+            store = Ext.create('GeoExt.data.store.Features', {features: coll});
+        });
+        afterEach(function() {
+            store = null;
+            coll = null;
+            feature = null;
+        });
+
+        it('is defined', function() {
+            expect(store.getByFeature).not.to.be(undefined);
+        });
+        it('returns the right feature record', function() {
+            expect(store.getByFeature(feature).getFeature()).to.be.equal(feature);
+        });
```

Comment:
What will be returned if the store is queried with a non managed `ol.Feature`?

```javascript
expect(store.getByFeature(new ol.Feature())).to.be // ??
expect(store.getByFeature(new ol.Feature()).getFeature()).to.be // ??
```

Can you add a test for this?